docker/0.3.0/fedimintd-mainnet-tls/.env

```diff
@@ -0,0 +1,10 @@
+# This domain should point to the machine fedimintd is being deployed to
```

Comment:
Does docker automatically pull in the `.env` file? 

I saw something that said it might need to be something like this: 

```
docker-compose -f docker-compose.yml --env-file .env up
```

If that's the case, might be useful to put at the top of this file as a quick note.

Reply:
I think it does? It did for me iirc.

Reply:
I was using `docker compose up` and not `docker-compose up` btw, if that makes a difference?